data/block/block_test.go

```diff
@@ -295,6 +362,160 @@ func TestBody_IntegrityAndValidityOK(t *testing.T) {
 	assert.Equal(t, nil, body.IntegrityAndValidity())
 }
 
+func TestBody_Clone(t *testing.T) {
+	t.Parallel()
+
+	txHash0 := []byte("txHash0")
+	mb0 := block.MiniBlock{
+		ReceiverShardID: 0,
+		SenderShardID:   0,
+		TxHashes:        [][]byte{txHash0},
+	}
+
+	body := &block.Body{}
+	body.MiniBlocks = append(body.MiniBlocks, &mb0)
+
+	clonedBody := body.Clone()
+
+	assert.True(t, reflect.DeepEqual(body, clonedBody))
+}
+
+func TestHeader_SetMiniBlockHeaderHandlersNilMiniBlockHeaderHandlers(t *testing.T) {
+	t.Parallel()
+
+	hdr := block.Header{}
+	err := hdr.SetMiniBlockHeaderHandlers(make([]data.MiniBlockHeaderHandler, 0))
+	assert.Nil(t, err)
+	assert.Nil(t, hdr.MiniBlockHeaders)
+}
+
+func TestHeader_SetMiniBlockHeaderHandlersWithError(t *testing.T) {
```

Comment:
add `t.Parallel()` at the beginning of this test. Otherwise, the `TestHeader_SetMiniBlockHeaderHandlersWithError` running will be sequentially and only the inner tests will be run in parallel

Reply:
done